lib/browsers/phantomjs-control-script.js

```diff
@@ -66,7 +66,8 @@ page.onInitialized = function (arg1) {
                 window.callPhantom({
                     name: 'slaveDispose'
                 });
-            }
+            },
+            replaceConsole: true
```

Comment:
I would change the name of the parameter to something like `localConsole: false`.
We will always have the local console in the browser, and the remote console on the server (the one pushed by websockets)